bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -22,16 +22,18 @@ local_setup() {
     try --max 48 --delay 5 running_nginx
 }
 
-@test 'shutdown, make a snapshot, and clear everything' {
+@test 'shutdown, make a snapshot, and run factory-reset' {
     rdctl shutdown
-    rdctl snapshot create "$SNAPSHOT"
+    snapshot_description="first snapshot"
+    rdctl snapshot create "$SNAPSHOT" --description "$snapshot_description"
     run rdctl snapshot list
```

Comment:
Separate test steps with newlines

Reply:
I generally am not big with a single-line block, but if you want each `rdctl` command in its own block, you'll get them at each end of this test

Reply:
I don't understand what you are saying here.

I want a newline before each `run` statement, to show that you are testing something else now:

```
run ...
assert
assert

run
assert
assert
...
```